synquacer.mk

```diff
@@ -46,10 +51,31 @@ EDK2_ARCH ?= AARCH64
 # Targets
 ################################################################################
 .PHONY: all
-all: edk2 optee-os tfa
+all: edk2 optee-os scp tfa
 
 .PHONY: clean
-clean: edk2-clean optee-os-clean tfa-clean
+clean: edk2-clean optee-os-clean scp-clean tfa-clean
+
+################################################################################
+# SCP
+################################################################################
+SCP_FLAGS ?= \
+	CC=$(AARCH32_NONE_CROSS_COMPILE)gcc \
+	PRODUCT=synquacer \
+	MODE=$(SCP_BUILD)
+
+.PHONY: scp
+scp: aarch32-none
```

Comment:
I think it's possible to add a `toolchains: aarch32-none` somewhere in this file instead.

Reply:
HNY Jens. 
OK. I will move the recipe to aarch32-none. 
I think aarch32-none should be in toolchain.mk in long run if other products start using it. 
But we can think of moving the aarch32-none recipe when that time comes.

Reply:
I agree with Jens. This whole block should be:
```make
toolchains: aarch32-none

.PHONY: scp
scp:
	$(MAKE) -C $(SCP_PATH) $(SCP_FLAGS) all
	# other commands...
```

Reply:
The devbox still depends on aarch64 to build other stuffs. 
Shouldn't the line contain both aarch32-none and aarch64? 
Otherwise , synquacer.mk will override the recipe and forget to download the aarch64 toolchain. 

```Makefile
toolchains: aarch32-none aarch64

.PHONY: scp
scp:
	$(MAKE) -C $(SCP_PATH) $(SCP_FLAGS) all
	# other commands...
```

Reply:
No, makefiles don't work like that. Please note that:
```make
a: b c
```
...is equivalent to:
```make
a: b
a: c
```
In other words, the second target line will not override the first, it will just add a dependency (`c`). Things would be different if the targets had recipes, indeed.

Reply:
Thanks for correction. It works as you said. 
Ok. I will tidy things up and push again.

Reply:
You chose to keep the dependency `scp: aarch32-none` which is technically correct, but not done for any other targets. That's fine.